torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -365,7 +399,12 @@ class CppMicroGemmFP32Vec(CppMicroGemm):
         if constexpr (accum) {
             constexpr int row = i / COLS;
             constexpr int col = i % COLS;
-            vc[i] = Vectorized::loadu(C + row * ldc + col * VLEN);
+            {%- if input_dtype in [torch.bfloat16, torch.float16] and input2_dtype == torch.int8 %}
+            auto c = VectorizedIn::loadu(C + row * ldc + col * VLEN, VLEN);
+            vc[i] = at::vec::convert<{{compute_t}}>(c);
```

Comment:
I guess this is related to the question why we use bf16/fp16 for C.

Reply:
I'll make it FP32. Then FP32 -> BF16/FP16 conversion will happen after micro-kernel compute, and may help boost perf because each index of `C` is written to/read from more than once in the micro-kernel.

Thanks!